crossbeam-skiplist/src/base.rs

```diff
@@ -227,6 +227,24 @@ impl<K, V> Node<K, V> {
         }
     }
 
+    /// Decrements the reference count of a node, pinning the thread and destoying the node
+    /// if the count become zero.
+    #[inline]
+    unsafe fn decremnt_with_pin<F>(&self, pin: F)
```

Comment:
Typo: `decremnt` -> `decrement`

Reply:
Oops, thank you. Fixed in https://github.com/crossbeam-rs/crossbeam/pull/337/commits/d07e0eb83bc33451b6587224467e8851982be103.